dair_pll/drake_utils.py

```diff
@@ -36,16 +41,27 @@
 from pydrake.symbolic import Expression  # type: ignore
 from pydrake.systems.analysis import Simulator  # type: ignore
 from pydrake.systems.framework import DiagramBuilder  # type: ignore
-from pydrake.systems.meshcat_visualizer import MeshcatVisualizer  # type: ignore
+from pydrake.visualization import VideoWriter  # type: ignore
 
 from dair_pll import state_space
+from dair_pll import file_utils
 
 WORLD_GROUND_PLANE_NAME = "world_ground_plane"
 DRAKE_MATERIAL_GROUP = 'material'
 DRAKE_FRICTION_PROPERTY = 'coulomb_friction'
 N_DRAKE_FLOATING_BODY_VELOCITIES = 6
 DEFAULT_DT = 1e-3
 
+GROUND_COLOR = np.array([0.5, 0.5, 0.5, 0.1])
+
+CAM_FOV = np.pi/5
+VIDEO_PIXELS = [480, 640]
+FPS = 30
+SENSOR_POSE = RigidTransform(RollPitchYaw([-np.pi/2, 0, np.pi/2]), [1, 0, 0.2])
+EXP_NAME = os.environ['PLL_EXPERIMENT']
+VIDEO_FILENAME = op.join(file_utils.temp_dir(
```

Comment:
Likely more appropriately placed in `file_utils.py`

Reply:
Done.